src/test/java/com/google/firebase/database/FirebaseDatabaseTest.java

```diff
@@ -44,13 +48,13 @@
           .setDatabaseUrl("https://firebase-db-test.firebaseio.com")
           .build();
 
-  @BeforeClass
-  public static void setupClass() {
+  @Before
```

Comment:
Made this change as many test methods in this class require a default FirebaseApp instance that is instantiated only once: this conflicted with my tests where I need to create a default instance and then clean up. So I changed this up to be on a per-method basis which is cleaner and prevents methods from conflicting.

Reply:
Thanks for doing this!